rules_engine/tasks.py

```diff
@@ -21,7 +21,9 @@
 from .models import RegexRule, IntervalRule, ExpirationRule
 
 
-@shared_task(task_serializer='pickle', result_serializer='pickle')
+@shared_task(
+    task_serializer='pickle', result_serializer='pickle', rate_limit='30/s',
```

Comment:
please check comments already written for rate_limit

Reply:
i cannot find the comment on rate_limit is but in my math 0.5tasks/second means 'execute a task every 2 seconds'. thus 100tasks / 0.5tasks_per_second=200 seconds. what is the problem with that?